Ejercicio3Poo/src/main/java/com/mycompany/ejercicio3poo/Calculadora.java

```diff
@@ -0,0 +1,72 @@
+package com.mycompany.ejercicio3poo;
+
+import java.util.Random;
+
+public class Calculadora {
+
+    public ModeloCalculadora dividir;
+
+    public int sumar(int numero1, int numero2, int numero3) {
+        int resultadoSuma = numero1 + numero2 + numero3;
+        return resultadoSuma;
+    }
+
+    public String restar(int numero1, int numero2){
+        int resultadoResta = numero1 - numero2;
+        String restar = "el resultado de la resta es : " + resultadoResta;
+        return restar;
+    }
+
+    public double multiplicar(int numero1, int numero2, int numero3, int numero4, int numero5) {
+        double resultadoMultiplicacion = (numero1 * numero2 * numero3 * numero4 * numero5);
+        return resultadoMultiplicacion;
+    }
+
+    public float division(ModeloCalculadora dividir ){
+        float resultado = dividir.getNumeroUno()/dividir.getNumeroDos();
+        return resultado;
+    }
+
+    public String potencia() {
+        Random numeroAleatorio = new Random();
+        int random = numeroAleatorio.nextInt(100);
+        int potencia = random ^ 2;
```

Comment:
You can use 

https://parzibyte.me/blog/2019/09/02/elevar-numero-java-math-pow/

In order to avoid complications and validations